spec/models/story_spec.rb

```diff
@@ -371,4 +371,20 @@ def fake_response(content, type, code = '200')
       expect(story_has_url_without_anchor.already_posted_recently?).to be true
     end
   end
+
+  describe "scopes" do
+    context "recent" do
+      before do
+        create(:story, title: "Front Page")
+        create(:story, title: "Front Page 2")
+
+        create(:story, title: "New Story", downvotes: 3) # downvotes simulate story off 'front'
+      end
+
+      it "returns the newest stories that have not yet reached the front page" do
+        expect(described_class.recent).to include described_class.find_by(title: "New Story")
```

Comment:
```suggestion
        expect(Story.recent).to include Story.find_by(title: "New Story")
```

I can't imagine implementing the `hottest` scope on anything else. (Same patch on the next line.)